src/ui/Misc/Slider.ts

```diff
@@ -756,15 +756,15 @@ class SliderGraph {
   }
 
   private padGraphWithEmptyData(data: ISliderGraphData[]) {
-    let oneStepOfGraph = data[0].end - data[0].start;
+    const oneStepOfGraph = data[0].end - data[0].start;
     this.padBeginningOfGraphWithEmptyData(data, oneStepOfGraph);
     this.padEndOfGraphWithEmptyData(data, oneStepOfGraph);
   }
 
   private padBeginningOfGraphWithEmptyData(data: ISliderGraphData[], oneStepOfGraph: number) {
     if (data[0].start > this.slider.options.start && data[0].start > oneStepOfGraph) {
       const difToFillAtStart = data[0].start - this.slider.options.start;
-      const nbOfStepsAtStart = Math.round(difToFillAtStart / oneStepOfGraph);
+      const nbOfStepsAtStart = Math.round(difToFillAtStart / (oneStepOfGraph || 1));
```

Comment:
Maybe we could also add a max value there. When the index create range buckets automatically, it's a bit "opaque" how the buckets are generated. It's unlikely, but it could be possible that we would have to fill "thousands" of steps, and it would be laggy.

Reply:
Would you suggest as max value? 1000?

Reply:
I saw there was a const MAX_NUMBER_OF_STEPS at 100 nvm